src/frontend/js/components/Dashboard/DashboardBreadcrumbs.tsx

```diff
@@ -0,0 +1,123 @@
+import { useMatches, useNavigate } from 'react-router-dom';
```

Comment:
This file module does too much things. Can you split it in separate modules ?

`BreadcrumbProvider`, `useBreadcrumbPlaceholders`,  `useBreadcrumbParts` then `DashboardBreadcrumbs` or at least `BreadcrumbProvider` and `DasbhoardBreadcrumbs`

Reply:
100% agreed